backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java

```diff
@@ -41,30 +37,4 @@
   public Option<BaseGitMacheteBranch> getBranchByName(String branchName) {
     return branchByName.get(branchName);
   }
-
-  @Override
-  public IGitRebaseParameters getParametersForRebaseOntoParent(BaseGitMacheteNonRootBranch branch)
```

Comment:
We didn't need the repository for anything even here in the removed impl, this method could've as well been static